BugsAndExploits/TntDuping.md

```diff
@@ -30,3 +25,9 @@ These specific blocks are able to create a normal update after the list of block
 That needs to happen before the TNT block is removed: before a block behind the tnt replaces it. if there is no blocks, it can be after the moving TNT was created.
 This update can also be propagated through other blocks, as long as it is instant (for example using rails, dust, noteblocks (1.13+), or TNT itself), which allows for up to 11 TNT to be duplicated using a single piston push, the 12th block being for updating.
 The tnt moving block is created from the moved block list regardless of the original tnt being ignited or not, and this means that the TNT is preserved in block form.
+
+## Mojang's Opposition
+Mojang historically opposes TNT Duping as they consider it an exploit, and they have to patch it four times, but in the 1.16 snapshots they declared they will leave it in the game for the time being, due to a lack of replacement for what TNT duping currently allows: [https://www.reddit.com/r/Minecraft/comments/fkt9jf/anchor_yourself_to_the_nether_snapshot_20w12a_is/fkv33q3](https://www.reddit.com/r/Minecraft/comments/fkt9jf/anchor_yourself_to_the_nether_snapshot_20w12a_is/fkv33q3)
```

Comment:
have tried to

Reply:
I would add this for more context: https://www.reddit.com/r/Minecraft/comments/e5gx0s/comment/f9q2eih/

Reply:
Done